.circleci/config.yml

```diff
@@ -1,17 +1,25 @@
 version: 2.1
 jobs:
-  Full build without itests:
-    docker: &image
-      - image: circleci/openjdk:11-jdk
+  FullBuild:
+    machine:
+      image: circleci/classic:latest
+      docker_layer_caching: true
     steps:
-      - setup_remote_docker:
-          docker_layer_caching: true
       - checkout
+      - run:
+          name: Install java 11
+          command: |
+            sudo add-apt-repository ppa:openjdk-r/ppa
+            sudo apt-get update -q
+            sudo apt install -y openjdk-11-jdk
+      - run:
+          name: Set default java 11
+          command:
+            sudo update-java-alternatives --set java-1.11.0-openjdk-amd64
       - run:
           name: Building
-          # Skip itests, which require another docker image and don't work in CircleCI
```

Comment:
❗️The Jenkinsfile should be removed because we no longer need it to run the itests

Reply:
All set - removed file